data/sql/updates/pending_db_world/rev_1550359453246256000.sql

```diff
@@ -0,0 +1,1090 @@
+-- Undercity Fight
+-- The Battle for the Undercity - Alliance
+SET @WRYNN := 32401;
+SET @JAINA := 32402;
+SET @BROLL := 32376;
+SET @VALEERA := 32378;
+SET @SWSOLDIER := 32387;
+SET @SWSOLDIER2 := 31639;
+SET @ALLIANCETANK := 31638;
+SET @ALLIANCEPLANE := 32388;
+SET @DEMOLISHER := 31652;
+SET @WSGUARD := 31739;
+SET @PLAGUEDFELBEAST := 32392;
+SET @GUARDIAN := 32390;
+SET @DREADLORD := 32391;
+SET @BLIGHTDOCTOR := 32397;
+SET @COLLABORATOR := 32396;
+SET @BETRAYER := 32394;
+SET @CHEMIST := 32395;
+SET @GENERATOR := 36212;
+SET @PUTRESS := 31530;
+SET @BLIGHTWORM := 32483;
+SET @KHANOK := 32511;
+SET @FAILEDEXPERIMENT := 32519;
+SET @THRALL := 32518;
+SET @SYLVANAS := 32365;
+SET @VARIMATHRAS := 31565;
+SET @SAURFANG := 32315;
+SET @VOICE := 32277;
+SET @VOLJIN := 31649;
+SET @SPELLPHASE128 := 60815;
+SET @SPELLPHASE64 := 59062;
+SET @BLIGHTBREATH := 61125;
+SET @INGEST := 61123;
+SET @SPELLMISSILES := 54900;
+SET @BLIGHTEMPOWERMENT := 59449;
+SET @WRYNNBUFF := 60964;
+SET @QUEST := 13377;
+SET @QUEST_H := 13267;
+SET @PORTAL := 193956;
+SET @CGUID := 3109763;
+SET @GGUID := 2134392;
+
+-- ALLIANCE EVENT
+UPDATE `creature_template` SET HealthModifier=915.547520661157 WHERE entry=32483;
+
+UPDATE `creature_template` SET InhabitType=4, minlevel=74, maxlevel=75 WHERE entry=@ALLIANCEPLANE;
+UPDATE `creature_template` SET minlevel=75, maxlevel=76 WHERE entry=@SWSOLDIER;
+
+UPDATE `creature_template` SET unit_flags=0, minlevel=74, maxlevel=76 WHERE entry=@PLAGUEDFELBEAST;
+UPDATE `creature_template` SET unit_flags=0, minlevel=74, maxlevel=76 WHERE entry=@GUARDIAN;
+UPDATE `creature_template` SET unit_flags=0, minlevel=74, maxlevel=76 WHERE entry=@DREADLORD;
+UPDATE `creature_template` SET unit_flags=0, minlevel=74, maxlevel=76 WHERE entry=@BLIGHTDOCTOR;
+UPDATE `creature_template` SET unit_flags=0, minlevel=74, maxlevel=76 WHERE entry=@COLLABORATOR;
+UPDATE `creature_template` SET unit_flags=0, minlevel=74, maxlevel=76 WHERE entry=@BETRAYER;
+UPDATE `creature_template` SET unit_flags=0, minlevel=74, maxlevel=76 WHERE entry=@CHEMIST;
```

Comment:
please always prefer compact queries, e.g. `UPDATE whatever SET something WHERE entry IN (....)` whenever updates/inserts/deletions can be grouped